src/main/java/com/ibm/crypto/plus/provider/OpenJCEPlusFIPS.java

```diff
@@ -103,9 +103,11 @@ public OpenJCEPlusFIPS() {
             debug.println("New OpenJCEPlusFIPS instance");
         }
 
-        if (!isPlatformSupported) {
-            throw new UnsupportedOperationException(
-                        "OpenJCEPlusFIPS is not supported on this non FIPS " + osName + " " + osArch + " platform");
+        if (!isFIPSCertifiedPlatform) {
+            System.out.println("WARNING: OpenJCEPlusFIPS is running in developer mode. Non production workload assumed. This environment is not certified for FIPS 140-3: " + osName + ":" + osArch);
```

Comment:
Do we need to remove this System.out.println(), since the same message will be printed when debug is enabled below?